server/graphql/loaders/index.js

```diff
@@ -591,21 +525,6 @@ export const loaders = req => {
       const transactions = await models.Transaction.findAll({ where, order });
       return sortResults(keys, transactions, 'OrderId', []);
     }),
-    findByOrderId: options =>
-      createDataLoaderWithOptions(
-        (OrderIds, options) => {
-          return models.Transaction.findAll({
-            where: {
-              OrderId: { [Op.in]: OrderIds },
-              ...options.where,
-            },
-            order: [['createdAt', 'DESC']],
-          }).then(results => sortResults(OrderIds, results, 'OrderId', []));
-        },
-        cache,
-        options,
-        'transactions',
-      ),
```

Comment:
There's already a `Transaction.byOrderId` above, and this one was not working since each call was instantiating a new `DataLoader`, defeating the purpose of using one.